addons/better-quoter/userscript.js

```diff
@@ -1,16 +1,160 @@
 export default async function ({ addon, global, console }) {
+  function getHTMLOfSelection() {
+    var range;
+    if (document.selection && document.selection.createRange) {
+      range = document.selection.createRange();
+      return range.htmlText;
+    } else if (window.getSelection) {
+      var selection = window.getSelection();
+      if (selection.rangeCount > 0) {
+        range = selection.getRangeAt(0);
+        var clonedSelection = range.cloneContents();
+        var div = document.createElement("div");
+        div.appendChild(clonedSelection);
+        return div.innerHTML;
```

Comment:
I'm not sure if this is actually true... but the way you are doing this can cause Cross Scripting if someone puts a script tag for example... I'm sure the chrome reviewers might now like that very much...